### ISSUE TITLE
freezing only a fix and specific set of connections between two dense layers during training

### ISSUE THREAD

**Issue by Viveksbawa** [fchollet/keras#6333]
I need to freeze a set of connection between two dense layers while training. Similar to dropout but instead of random, we have a specific set of connection that should be frozen for all the training epochs.Can anyone suggest any way to implement this with keras ?
